sdk/digitaltwins/Azure.DigitalTwins.Core/src/Models/BasicDigitalTwinPropertyMetadata.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json.Serialization;
+
+namespace Azure.DigitalTwins.Core
+{
+    /// <summary>
+    /// Contain metadata about changes on properties on a digital twin or component.
+    /// </summary>
+    public class BasicDigitalTwinPropertyMetadata
+    {
+        /// <summary>
+        /// The date and time the property was last updated.
+        /// </summary>
+        [JsonPropertyName(DigitalTwinsJsonPropertyNames.MetadataPropertyLastUpdateTime)]
+        public DateTimeOffset LastUpdatedOn { get; set; }
```

Comment:
Why not LastUpdateTime? Did we pick a bad property name in the service?

Reply:
Azure SDK board prefers we use this style of naming for timestamp fields. For instance, we use "UploadedOn" rather than "UploadTime" in the digital twins model data class

Reply:
Even if that doesn't align with the service API? Is this language specific?

Reply:
Also - is there a recommendation on the MetadataPropertyLastUpdateTime naming? Should the name of that constant align with the name used by the service, or the name used in the projected API?

Reply:
Yes, the Azure SDK board says they prefer the suffix On for date/time properties.